core/design-system/src/main/kotlin/team/noweekend/core/design/system/core/component/control/page/NWKPageControl.kt

```diff
@@ -0,0 +1,92 @@
+package team.noweekend.core.design.system.core.component.control.page
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CornerBasedShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import team.noweekend.core.design.system.core.component.control.page.defaults.NWKPageControlColors
+import team.noweekend.core.design.system.core.component.control.page.defaults.NWKPageControlDefaults
+import team.noweekend.core.design.system.foundation.theme.NWKTheme
+
+@Composable
+fun NWKPageControl(
+    pageSize: Int,
+    currentPosition: Int,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(
+                horizontal = NWKTheme.spacing.space150,
+                vertical = NWKTheme.spacing.space100,
+            ),
+        horizontalArrangement = Arrangement.spacedBy(
+            space = NWKPageControlDefaults.indicatorGap,
+            alignment = Alignment.CenterHorizontally,
+        ),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        repeat(pageSize) { page ->
+            NWKDotIndicator(
+                selected = page == currentPosition,
+                modifier = Modifier,
+            )
+        }
+    }
+}
+
+@Composable
+private fun NWKDotIndicator(
+    selected: Boolean,
+    modifier: Modifier = Modifier,
+    colors: NWKPageControlColors = NWKPageControlDefaults.indicatorColor(),
+    shape: CornerBasedShape = NWKPageControlDefaults.indicatorShape,
+) {
+    val backgroundColor: Color by colors.backgroundColor(selected)
+    val shapeModifier = when {
+        selected -> {
+            modifier.size(
+                width = NWKPageControlDefaults.IndicatorType.SELECTED.width,
+                height = NWKPageControlDefaults.IndicatorType.SELECTED.height,
+            )
+        }
+
+        else -> {
+            modifier.size(
+                width = NWKPageControlDefaults.IndicatorType.DEFAULT.width,
+                height = NWKPageControlDefaults.IndicatorType.DEFAULT.height,
+            )
+        }
+    }
+
+    Box(
+        modifier = modifier
+            .then(shapeModifier)
+            .clip(shape = shape)
+            .background(color = backgroundColor),
+    )
+}
```

Comment:
p3: IndicatorType만 분기처리 해서 사용하는 것이 중복 코드도 줄이고 modifier를 반환하지 않는게 깔끔해 보여서 한번 제안 해볼게! 타입이 늘어난다면 when으로 처리해도 괜찮을거 같기도 하고~? 
```suggestion
val indicatorType = if (selected) {
    NWKPageControlDefaults.IndicatorType.SELECTED
} else {
    NWKPageControlDefaults.IndicatorType.DEFAULT
}

Box(
    modifier = modifier
        .size(
            width = indicatorType.width,
            height = indicatorType.height,
            )
        .clip(shape)
        .background(backgroundColor)
)
```

Reply:
@SEO-J17 좋네?

Reply:
@SEO-J17 아 요고는 안그래도 문제 있어보여서 같이 수정해볼게! 💋

Reply:
[b5587b5](https://github.com/mash-up-kr/NoWeekend-Android/pull/44/commits/b5587b5e4d6d49771449b730d1178e6a93cb5b8b) 반영 완료